beacon-chain/sync/rpc_ping.go

```diff
@@ -94,7 +94,7 @@ func (r *Service) validateSequenceNum(seq uint64, id peer.ID) (bool, error) {
 		return false, err
 	}
 	if md == nil {
-		return false, nil
+		return true, nil
```

Comment:
would expect the bool value to be the opposite 
```
func (r *Service) validateSequenceNum(seq uint64, id peer.ID) (bool, error) {
	md, err := r.p2p.Peers().Metadata(id)
	if err != nil {
		return **false**, err
	}
	if md == nil {
		return **false**, nil
	}
	if md.SeqNumber != seq {
		return **false**, nil
	}
	return **true**, nil
}

Reply:
ok done